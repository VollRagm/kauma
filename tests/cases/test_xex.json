{
    "testcases": {
        "b856d760-023d-4b00-bad2-15d2b6da22fe": {
            "action": "xex",
            "arguments": {
                "mode": "encrypt",
                "key": "B1ygNO/CyRYIUYhTSgoUysX5Y/wWLi4UiWaVeloUWs0=",
                "tweak": "6VXORr+YYHrd2nVe0OlA+Q==",
                "input": "/aOg4jMocLkBLkDLgkHYtFKc2L9jjyd2WXSSyxXQikpMY9ZRnsJE76e9dW9olZIW"
            }
        },
        "254eaee7-05fd-4e0d-8292-9b658a852245": {
            "action": "xex",
            "arguments": {
                "mode": "decrypt",
                "key": "B1ygNO/CyRYIUYhTSgoUysX5Y/wWLi4UiWaVeloUWs0=",
                "tweak": "6VXORr+YYHrd2nVe0OlA+Q==",
                "input": "lr/ItaYGFXCtHhdPndE65yg7u/GIdM9wscABiiFOUH2Sbyc2UFMlIRSMnZrYCW1a"
            }
        }
    }
}