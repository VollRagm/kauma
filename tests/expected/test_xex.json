{
    "responses": {
        "b856d760-023d-4b00-bad2-15d2b6da22fe": {
            "output": "mHAVhRCKPAPx0BcufG5BZ4+/CbneMV/gRvqK5rtLe0OJgpDU5iT7z2P0R7gEeRDO"
        },
        "254eaee7-05fd-4e0d-8292-9b658a852245": {
            "output": "SGV5IHdpZSBrcmFzcyBkYXMgZnVua3Rpb25pZXJ0IGphIG9mZmVuYmFyIGVjaHQu"
        }
    }
}